back-office/wishlist.md

```diff
@@ -0,0 +1,337 @@
+
+-	a new list that can be created by clicking Create New wishlist
+![wishlist screen 3](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-3.png)
+
+The user can only select one list.  
+
+What happens in each case?
+1.	When no list has been created: the logged-in user has the option to add the item(s) to the default list or create a new list  
+2.	When at least one list has been created: the logged-in user has the option to add the item(s) to the default favorites list, their existing lists or a new list.
+3.	When more than X lists have been created: the logged-in user can scroll down to see all the lists created.
+
+There is no list limit
+
+The heart is filled in once the product is added to a favorites list  
+
+Favorites heart for combinations: in the list pages, the heart will not be filled in if the default combination is not selected
+![wishlist screen 4](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-4.png)
+
+If the user clicks on the filled-in heart again, the product is removed from the wishlist. If the product is deleted by the seller, it should no longer appear in the users' favorites lists.
+
+For a non-logged-in user:
+For a non-logged-in user: when they click on the heart, a login pop-up opens informing them that they need to be logged in to add a product to their favorites:
+![wishlist screen 5](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-5.png)
+
+
+When clicking on "Sign in", the user is redirected to the login page. Once the connection is made, the user is directly redirected to the page they were on.
+
+Special login pop-up features:
+-	add a go-back setting to the URL so that, once the user is logged in, they are redirected to the page they were on (any page)
+-	the user is then asked to select the favorites list in which he/she wants to save the product.
+
+### US 2: View my favorite products  
+##### As a logged-in user, I can view and sort my favorite products
+
+Functional rules:  
+What type of users can add products to their favorites?
+A logged-in user
+
+For non-logged-in users:  
+Non-logged-in users cannot access their favorites list  
+
+For logged-in users:  
+As a logged-in user, I can access my favorite products by:  
+- Going to my account  
+- Clicking on "Wishlists"
+![wishlist screen 6](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-6.png)
+
+- I come to the page with all my favorites lists (default one + created lists)
+![wishlist screen 7](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-7.png)
+
+- When I click on one of the lists, it brings me to a page that lists my previously added products  
+
+- The number of products in the list is displayed to the left of the list name  
+This means that it accounts for all the idproduct-idproduct allocated
+For example, if there are 3 variations of the same product it counts as 3 products.
```

Comment:
```suggestion
For example, if there are 3 combinations of the same product it counts as 3 products.
```
Are you sure ? Because a product with different combinations has the same id for each combination